app/survey/components/graphs/AffordFairhold.tsx

```diff
@@ -1,9 +1,10 @@
 import React from "react"
-import { BarOrPieResults } from "@/app/survey/types";
 import SurveyGraphCard from "@/app/survey/components/SurveyGraphCard";
 import { PieChart, Pie, Legend, ResponsiveContainer } from "recharts";
+import { useSurveyContext } from "../../context";
+export const AffordFairhold = () => {
+    const affordFairhold = useSurveyContext().barOrPie.affordFairhold;
```

Comment:
You can also contextually rename variables this way as well.

```suggestion
    const { affordFairhold: data } = useSurveyContext().barOrPie;
```